reviewing/dependencies-hierarchy/src/getTree.ts

```diff
@@ -80,7 +80,7 @@ function getTreeHelper (
   const peers = new Set(Object.keys(opts.currentPackages[parentId].peerDependencies ?? {}))
 
   const resultDependencies: PackageNode[] = []
-  let resultHeight = 0
+  let resultHeight: number | null = null
```

Comment:
In a debugger, I noticed a test package with no dependencies return a height of `0` from `getTree`. By the comment in `DependencyInfo`, this should instead be `null` since the dependencies array is empty.

https://github.com/pnpm/pnpm/blob/7eb7056c92133bcba7989aa94a020ba4303df2eb/reviewing/dependencies-hierarchy/src/getTree.ts#L33-L37

It turns out this is the case when `deps == null`. The `getTree` function early returns with a `null` height:

https://github.com/pnpm/pnpm/blob/7eb7056c92133bcba7989aa94a020ba4303df2eb/reviewing/dependencies-hierarchy/src/getTree.ts#L71-L73

However, when `deps` is `{}`, `resultHeight` gets initialized to `0`. The subsequent `.forEach` loop then iterates over the non-empty object.